packages/augur-ui/src/modules/trading/components/form.tsx

```diff
@@ -1183,6 +1185,7 @@ class Form extends Component<FromProps, FormState> {
                     )}
                     <SquareDropdown
                       defaultValue={s.expirationDateOption}
+                      openTop={!(showTip && !initialLiquidity)}
```

Comment:
I don't see how `showTip` and dropdown correspond. It's confusing. 

showTip is to show a tip message at the bottom of the order form not a height. Use a different approach to determine if the dropdown will not be visible when opened.